Resources/public/Admin.js

```diff
@@ -329,6 +329,11 @@ var Admin = {
 
     setup_collection_buttons: function(subject) {
 
+        // Remove previously attached event(s) which causing function to be called multiple times
+        jQuery(subject).on('click', '.sonata-collection-add', function(event){
+            return false;
+        });
+
```

Comment:
I can't remove previously attached events with ".off" or "unbind". this is the only solution worked and found something similar on stackoverflow